pkg/webhook/pod/pod_mutater.go

```diff
@@ -114,3 +112,18 @@ func (pc *PodAdmissionControl) tikvHotRegionSchedule(tc *v1alpha1.TidbCluster, p
 	}
 	return nil
 }
+
+func (pc *PodAdmissionControl) getTikvConfigMap(tc *v1alpha1.TidbCluster) (*corev1.ConfigMap, error) {
+	selector, err := label.New().Instance(tc.Name).TiKV().Selector()
+	if err != nil {
+		return nil, err
+	}
+	cmlist, err := pc.kubeCli.CoreV1().ConfigMaps(tc.Namespace).List(metav1.ListOptions{LabelSelector: selector.String()})
+	if err != nil {
+		return nil, err
+	}
+	if len(cmlist.Items) < 1 {
+		return nil, fmt.Errorf("tc[%s/%s]'s tikv configmap failed to find", tc.Namespace, tc.Name)
+	}
+	return &cmlist.Items[0], nil
```

Comment:
After several updates of the config, there will be several configmaps for tikv, how can you make sure items[0] is the latest valid config?

Reply:
good catch, currently I get the configmap name by the coming pod template.